crates/sui-json-rpc/src/lib.rs

```diff
@@ -101,35 +92,6 @@ impl JsonRpcServerBuilder {
         Ok(self.module.merge(module.rpc())?)
     }
 
-    fn cors() -> Result<CorsLayer, Error> {
-        let acl = match env::var("ACCESS_CONTROL_ALLOW_ORIGIN") {
-            Ok(value) => {
-                let allow_hosts = value
-                    .split(',')
-                    .map(HeaderValue::from_str)
-                    .collect::<Result<Vec<_>, _>>()?;
-                AllowOrigin::list(allow_hosts)
-            }
-            _ => AllowOrigin::any(),
-        };
```

Comment:
Do we want to preserve this functionality by moving it to the new `sui-node` CORS layer?